telebot/types.py

```diff
@@ -5792,18 +5792,26 @@ class InputMediaPhoto(InputMedia):
         instead of parse_mode
     :type caption_entities: :obj:`list` of :class:`telebot.types.MessageEntity`
 
+    :param has_spoiler: Optional. True, if the uploaded photo is a spoiler
+    :type has_spoiler: :obj:`bool`
+
     :return: Instance of the class
     :rtype: :class:`telebot.types.InputMediaPhoto`
     """
-    def __init__(self, media, caption=None, parse_mode=None, caption_entities=None):
+    def __init__(self, media, caption=None, parse_mode=None, caption_entities=None, has_spoiler=None):
         if util.is_pil_image(media):
             media = util.pil_image_to_file(media)
     
         super(InputMediaPhoto, self).__init__(
             type="photo", media=media, caption=caption, parse_mode=parse_mode, caption_entities=caption_entities)
 
+        self.has_spoiler: Optional[bool] = has_spoiler
+
     def to_dict(self):
-        return super(InputMediaPhoto, self).to_dict()
+        ret = super(InputMediaPhoto, self).to_dict()
+        if self.has_spoiler:
```

Comment:
I propose to use "is not None" like in sendXXX methods. It will allow to set "False" obviously.